GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -402,8 +359,28 @@ private static string GetRegistryValue(RegistryKey root, string subkey, string k
             return value ?? string.Empty;
         }
 
-        private static string GetVisualStudioPath()
+        private static string GetVsDiffMergePath()
         {
+            //%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe
+            var vsVersionNumber = new string[] { "2020", "2019", "2018", "2017" };
```

Comment:
I'm afraid this is not correct to assume this. The history paint a different picture - 2003, 2005, 2008, 2010, 2012, 2015, 2017... 
MS has made the same move with VS2017 as they did with W10 - SAAS. So it is quite likely that we won't see another version of VS.